root/inc/breadcrumbs.html

```diff
@@ -48,8 +48,8 @@
       IF have_released; %>
     <a class="latest" href="<% IF module %>/pod/<% module.documentation %><% ELSE %>/release/<% release.distribution; END %>" title="<%- IF release.maturity == 'developer'; 'dev release, '; END %>go to latest"><span class="fa fa-step-forward"></span></a>
   <%- END; END; %>
-  <div class="inline"><% INCLUDE inc/river-gauge.html, distribution = release.distribution %></div>
-  <div class="inline"><%- INCLUDE inc/favorite.html %></div>
```

Comment:
The reason for removing the `inline` class in an otherwise unrelated change isn't mentioned.  From the face of it, it seems like it would break https://github.com/metacpan/metacpan-web/blob/d3f34a1/root/static/js/cpan.js#L301 (although the correctness and provenance of that line seems dubious at first glance).

Reply:
The div serves no real purpose, and then uses a silly class like "inline" to fix it.  I'll fix the javascript to handle this properly.

Reply:
I totally agree it's a stupid bit of markup. :-)